crates/sui-protocol-config/src/lib.rs

```diff
@@ -2094,6 +2114,10 @@ impl ProtocolConfig {
                 44 => {
                     // Enable consensus digest in consensus commit prologue on all networks..
                     cfg.feature_flags.include_consensus_digest_in_prologue = true;
+
+                    cfg.feature_flags.enable_soft_bundle = true;
+                    cfg.soft_bundle_max_num_certificates = Some(4);
+                    cfg.soft_bundle_max_serialized_size_bytes = Some(64 * 1024);
```

Comment:
We already have transaction size limits, so I'd be pretty comfortable just relying on those limits instead of having a separate limit. Otherwise it may create an awkward situation where some transactions are too large to go in a soft bundle for no clear reason.